feed.php

```diff
@@ -509,9 +514,9 @@ function rssListNamespace($opt) {
  * @author Andreas Gohr <andi@splitbrain.org>
  */
 function rssSearch($opt) {
-    if(!$opt['search_query']) return array();
+    if(!$opt['search_query'] || !actionOK('search')) return [];
```

Comment:
Here is the actual change (fix) - we just return empty results when someone is trying to search whenever it's disabled.

Reply:
Tip: next time separate the style adjustments and the functional change into two separate commits.